test/generated/testproto/test3_pb2.pyi

```diff
@@ -117,13 +117,13 @@ class SimpleProto3(google.protobuf.message.Message):
     @property
     def bool(self) -> global___OuterMessage3: ...
     OuterEnum: global___OuterEnum.V = ...
-    """Test having fieldname match messagename"""
+    r"""Test having fieldname match messagename"""
```

Comment:
Not a docstring, it will be 'mostly' ignored by the compile and IDEs, in the best case.

Reply:
These get processed by docutils (and therefore by some IDEs, PyCharm for example) as per ([PEP-258>Attribute Docstrings](https://www.python.org/dev/peps/pep-0258/#id40) in a way that they wouldn't otherwise if they weren't strings.

Reply:
I confirmed before adding these originally that PyCharm does in fact process these per PEP-258 as an Attribute Docstring.